### ISSUE TITLE
BUG: No kwargs in `df.apply(raw=True)`

### ISSUE THREAD

**Issue by erezinman** [pandas-dev/pandas#55753]
### Pandas version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://pandas.pydata.org/docs/whatsnew/index.html) of pandas.

- [ ] I have confirmed this bug exists on the [main branch](https://pandas.pydata.org/docs/dev/getting_started/install.html#installing-the-development-version-of-pandas) of pandas.

### Reproducible Example

```python
import pandas as pd

def func(a, b):
    return a + b

df = pd.DataFrame([[1, 2], [3, 4]])
df.apply(func, b=1) # OK
df.apply(func, raw=True, b=1) # TypeError: func() missing 1 required positional argument: 'b'
```

### Issue Description

`df.apply(raw=True)` should act similarly to `df.apply(Raw=False)`. It worked below v2.1.

### Installed Versions

<details>

INSTALLED VERSIONS
------------------
commit              : a60ad39b4a9febdea9a59d602dad44b1538b0ea5
python              : 3.9.18.final.0
python-bits         : 64
OS                  : Linux
OS-release          : 5.15.0-75-generic
Version             : #82~20.04.1-Ubuntu SMP Wed Jun 7 19:37:37 UTC 2023
machine             : x86_64
processor           : x86_64
byteorder           : little
LC_ALL              : None
LANG                : en_US.UTF-8
LOCALE              : en_US.UTF-8
pandas              : 2.1.2
numpy               : 1.25.2
pytz                : 2023.3.post1
dateutil            : 2.8.2
setuptools          : 67.4.0
pip                 : 23.0.1
Cython              : 3.0.4
pytest              : 7.4.3
hypothesis          : None
sphinx              : None
blosc               : None
feather             : None
xlsxwriter          : None
lxml.etree          : None
html5lib            : None
pymysql             : None
psycopg2            : None
jinja2              : None
IPython             : None
pandas_datareader   : None
bs4                 : None
bottleneck          : None
dataframe-api-compat: None
fastparquet         : None
fsspec              : None
gcsfs               : None
matplotlib          : None
numba               : None
numexpr             : 2.8.7
odfpy               : None
openpyxl            : None
pandas_gbq          : None
pyarrow             : None
pyreadstat          : None
pyxlsb              : None
s3fs                : None
scipy               : None
sqlalchemy          : None
tables              : None
tabulate            : None
xarray              : None
xlrd                : None
zstandard           : None
tzdata              : 2023.3
qtpy                : None
pyqt5               : None

</details>

**Comment by hvsesha** [comment]
hi @erezinman 

in updated pandas version  i am not getting error 

>>> import pandas as pd
>>> def func(a, b):
     return a + b

>>> df = pd.DataFrame([[1, 2], [3, 4]])
>>> a=df.apply(func, b=2) # OK
>>> print(a)
   0  1
0  3  4
1  5  6
>>> df.apply(func, raw=True, b=1)
   0  1
0  2  3
1  4  5
>>> pd.__version__
'2.2.0.dev0+360.g4556f11c46.dirty'
>>>

**Comment by Dhayanidhi-M** [comment]
Hi ,

In existing version , we have the below code which does not have the argument of Kwargs.

 result = np.apply_along_axis(
                wrap_function(self.func),
                self.axis,
                self.values
            )

I think , the upcoming pandas release has the fix for this issue by adding the kwargs as an argument.

 result = np.apply_along_axis(
                wrap_function(self.func),
                self.axis,
                self.values,
                *self.args,
                **self.kwargs,
            )

**Comment by lithomas1** [comment]
Tentatively marking this as a regression.

If someone wants to try and backport a fix for 2.1.x, I'm happy to merge that.

**Comment by dannyi96** [comment]
take

**Comment by dannyi96** [comment]
looks like this issue has been fixed in upcoming release as part of #55089
